include/ppx/window.h

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ppx_window_h
+#define ppx_window_h
+
+#include "ppx/config.h"
+
+#include <cstdint>
+
+namespace ppx {
+
+namespace grfx {
+struct SurfaceCreateInfo;
+}
+
+class Application;
+
+class Window
+{
+public:
+    struct WindowSize
+    {
+        uint32_t w, h;
```

Comment:
Only one variable declaration per line.
Use descriptive variable names and not single letter variable names.

Reply:
Fixed, and moved to #136